_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,238 @@
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [Neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+New contributor shoutout: **@ckelsel** has been responsible for porting a large number of [vim patches](https://github.com/neovim/neovim/commits?author=ckelsel), as has [**@lonerover**](https://github.com/neovim/neovim/commits?author=lonerover). Thank you!
+
+Old contributor shoutout: **@oni-link** isn't a name you read a lot in the commit logs, but he's been very helpful in reviewing changes and improving complicated PR's for quite some time. Thank you!
+
+
+What's new
+----------
+
+The [last newsletter](https://neovim.io/news/2016/11/) has gathered some dust,
+so we want to bring you up-to-date in the Neovim world. Neovim has had 3! releases since back them, so let us recap those first and foremost:
+
+### Release 0.2
+
+This release brings a host of fixes and improvements. We'll look at some of them closer below, but be sure to also check
+out the [release page](https://github.com/neovim/neovim/releases/tag/v0.2.0) for all the gory details.
+
+
+#### 0.2 for Users
+ - Microsoft Windows is now fully supported
+   - A working terminal mode is availabe in 0.2.1-dev, see below!
+ - Starting this release, neovim offers pre-built packages for [macOS](https://github.com/neovim/neovim/releases/download/v0.2.0/nvim-macos.tar.gz) and windows ([32 bit](https://github.com/neovim/neovim/releases/download/v0.2.0/nvim-win32.zip) and [64 bit](https://github.com/neovim/neovim/releases/download/v0.2.0/nvim-win64.zip)). The windows package contains a GUI and some additional tools like curl.exe.
+ - `:terminal` has seen various improvements (**@justinmk**, https://github.com/neovim/neovim/pull/6185)
+   - `findfile`, `:find` and `gf` now work from the terminal (**@tweekmonster**, https://github.com/neovim/neovim/pull/6009)
+   - Shell output is now throttled to maintain terminal responsiveness (**@justinmk**, https://github.com/neovim/neovim/pull/5396)
+    - Arguments for shell commands now work properly with `:term` (**@friedsock**, https://github.com/neovim/neovim/pull/4504)
+ - Numerous improvements to man.vim have been made(**@m-wynn**, **@raichoo**, **@renstrom**, **@justinmk**, https://github.com/neovim/neovim/pull/6693,  https://github.com/neovim/neovim/pull/6766, https://github.com/neovim/neovim/pull/6815, https://github.com/neovim/neovim/pull/3104, https://github.com/neovim/neovim/pull/7405)
+ - Man and help pages now contain a TOC (**@tweekmonster**, https://github.com/neovim/neovim/pull/5169)
```

Comment:
Maybe use 

```markdown
Man and help pages now contain a TOC (**@tweekmonster**, [neovim/neovim#5169](https://github.com/neovim/neovim/pull/5169))
```

Man and help pages now contain a TOC (**@tweekmonster**, [neovim/neovim#5169](https://github.com/neovim/neovim/pull/5169))

just make text looks nice, same as below.

Reply:
Good suggestion, thanks. I went with the ultra-short form.